menu.c

```diff
@@ -96,6 +96,12 @@ char DirEntryInfo[MAXDIRENTRIES][5]; // disk number info of dir entries
 char DiskInfo[5]; // disk number info of selected entry
 
 
+extern char minimig_ver_beta;
+extern char minimig_ver_major;
+extern char minimig_ver_minor;
+extern char minimig_ver_minion;
+char rtl_ver[20];
```

Comment:
It would be better to use the s[] buffer for it to not consume more RAM.

Reply:
New commit.